markdown/developers/guidelines/requirements/docker.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Docker Support
+subtitle: Software must be bundled using Docker and versioned.
+menu:
+  main:
+    weight: 70
+---
+
+Pipelines must have all software bundled using [Docker](https://www.docker.com/) - that is, it must be possible to run the pipeline with `-profile docker` and have all software requirements satisfied.
+
```

Comment:
```suggestion

We highly recommend using the docker images from the [biocontainers](https://biocontainers.pro], as these automatically come with corresponding singularity containers.

```

Reply:
I guess it makes sense to mention this here too.

Reply:
Yup, can make it even stronger I think

```suggestion

Tools should use docker images from [biocontainers](https://biocontainers.pro) where possible, as using Bioconda / Biocontainers gives support for conda + docker + singularity.

```